backend/review/management/commands/mergeinstructors.py

```diff
@@ -0,0 +1,185 @@
+    :param stat: Function to collect statistics.
+    """
+    for instructor_set in tqdm(duplicate_instructor_groups):
+        # Find a primary instance in the duplicate set. This should be the instance that is most
+        # "complete" -- in the case of instructors, this means that there is a linked user object.
+        potential_primary = [inst for inst in instructor_set if inst.user is not None]
+        # If no instance has a linked user, just pick one instructor.
+        if len(potential_primary) == 0:
+            stat(INSTRUCTORS_KEPT, 1)
+            primary_instructor = instructor_set[0]
+        # If there's only one user with a linked user object, select that as the primary.
+        # This should be the case that is hit most often.
+        elif len(potential_primary) == 1:
+            stat(INSTRUCTORS_KEPT, 1)
+            primary_instructor = potential_primary[0]
+        else:
+            # If all potential primary rows relate to the same user (with the same pk),
+            # go ahead and choose one arbitrarily.
+            if len(set([inst.user.pk for inst in potential_primary])) == 1 or force:
+                stat(INSTRUCTORS_KEPT, 1)
+                primary_instructor = potential_primary[0]
+            # Otherwise, we don't have enough information to merge automatically. There are multiple
+            # instructors marked as duplicates, but they link to different users and so could be
+            # different people. Report the PKs of these instructors in the statistics, but don't
+            # merge.
+            else:
+                stat(INSTRUCTORS_KEPT, len(instructor_set))
+                stat(INSTRUCTORS_UNMERGED, element=[i.pk for i in instructor_set])
+                # We don't have enough information to do this merge.
+                continue
+
+        # Filter for all instructors that aren't the primary.
+        duplicate_instructors = [
+            inst for inst in instructor_set if inst.pk != primary_instructor.pk
+        ]
+        # Transfer the sections and reviews of all non-primary instances to the primary instance.
+        for duplicate_instructor in duplicate_instructors:
+            sections = Section.objects.filter(instructors=duplicate_instructor)
+            for section in sections:
+                stat(SECTIONS_MODIFIED, 1)
+                if not dry_run:
+                    section.instructors.remove(duplicate_instructor)
+                    section.instructors.add(primary_instructor)
+
+            reviews = Review.objects.filter(instructor=duplicate_instructor)
+            for review in reviews:
+                stat(REVIEWS_MODIFIED, 1)
+                if not dry_run:
+                    review.instructor = primary_instructor
+                    review.save()
+
+            stat(INSTRUCTORS_REMOVED, 1)
+            if not dry_run:
+                duplicate_instructor.delete()
+
+
+# Middle Initial ([\w ']+)([a-zA-Z]+\.)([\w ']+)
```

Comment:
what's this comment mean

Reply:
This is a to-do to add a merge strategy that will ignore middle initials in merges

Reply:
Can you add TODO: to the front of the comment, in case someone forgets about it now but down the line is looking for TODOs and finds it